doc/nrf/installation/recommended_versions.rst

```diff
@@ -100,6 +62,80 @@ Zephyr features only available on Linux
   * Native Port (native_posix)
   * BabbleSim
 
+Supported operating systems for tools
+=====================================
+
+The following table shows the operating system versions that support the |NCS| tools:
+
+.. os_table_start
+
+.. list-table::
+  :header-rows: 1
+
+  * - Operating System
+    - x86
+    - x64
+    - ARM64
+  * - `Windows 11`_
+    - Tier 3
+    - Tier 3
+    - Not supported
+  * - `Windows 10`_
+    - Tier 3
+    - Tier 1
+    - Not supported
+  * - `Linux - Ubuntu 24.04 LTS`_
+    - Not supported
+    - Tier 2
+    - Not supported
+  * - `Linux - Ubuntu 22.04 LTS`_
+    - Not supported
+    - Tier 1
+    - Not supported
+  * - `Linux - Ubuntu 20.04 LTS`_
+    - Not supported
+    - Tier 2
+    - Not supported
+  * - `macOS 15`_
+    - Not applicable
+    - Tier 3
+    - Tier 3
+  * - `macOS 14`_
+    - Not applicable
+    - Tier 3
+    - Tier 3
+  * - `macOS 13`_
+    - Not applicable
+    - Tier 1
+    - Tier 1
+
+.. os_table_end
+
+Supported operating systems for firmware
+========================================
+
+The following table shows the operating system versions that support the |NCS| firmware:
+
+.. list-table::
+  :header-rows: 1
+
+  * - Operating System
+    - x86
+    - x64
+    - ARM64
+  * - `Windows 11`_
+    - Tier 3
+    - Tier 3
+    - Not supported
+  * - `Linux - Ubuntu 22.04 LTS`_
+    - Not supported
+    - Tier 1
+    - Not supported
+  * - `macOS 14`_
+    - Not applicable
+    - Tier 3
+    - Tier 3
```

Comment:
@karhama , bump: can you have a look at this table, as requested by @shanthanordic ? Thanks.

Reply:
I added comments based on what OS versions we have in CI side.

Reply:
Checked via chat about this table specifically, got the following answer from @karhama :

> Firmware table looks ok to me as is.